data/explorations/all_interactions/all_interactions.yaml

```diff
@@ -76,6 +76,44 @@ states:
       id: Continue
       solution: null
     param_changes: []
+  Drag And Drop Sort:
```

Comment:
Wasn't this part of a recent PR? Where you enabled interaction.

Reply:
Yes, it was previous one.